blog/2024-01-15-statecharts-whats-in-a-name/index.mdx

```diff
@@ -0,0 +1,112 @@
+
+### Consistency is Key
+
+You can do whatever you want (remember, “Up to you”) but whatever you decide, try to be consistent. Readers will learn the convention you choose and they’ll be able to rely on it. If you name states “Waiting”, “Downloading”, “Synchronizing” then when they see another word ending in “ing” they’ll probably assume it’s also a state. If you name entry actions and exit actions on states as well as actions on transitions all with camelCase then it’s clear that these are actions regardless of what they’re attached to. So remember, consistency is key and in an XState config object, most of these names end up exported as actual object keys!
+
+### Conventions and maintenance
+
+From consistency, conventions often emerge. These could be conventions adhered to across a statechart, an entire project or across all of the projects of a particular user or team. This makes the work easier to understand and easier to maintain, thereby streamlining collaboration.
+
+### Focusing on more important issues
+
+In the code world, developers used to fight about all sorts of crucial issues like line length, how to format ternary statements, etc. But then Prettier came along and there was nothing left to debate other than how to configure Prettier. In earlier days, importing XState machines into the Studio at times resembled a pile of shapes begging to be manually laid out. But we’ve made significant improvements to the Studio editor’s layout algorithm so that flows much better and there’s less dragging needed on the part of users. Conventions are important but there are more urgent tasks than sweating little details even if you still desire the benefits of consistent and intentional style.
+
+### Introducing Stately Style
+
+In order to help you put your debates and tired typing fingers to rest, we’re introducing a small but useful feature to help you improve your statecharts and keep on building your software in style! Think of this like the Prettier of XState and statecharts, all within [Stately Studio](https://stately.ai/studio).
+
+<p>
+  <ThemedImage
+    alt="Connect a project to a GitHub repository by clicking the Connect GitHub button."
+    sources={{
+      light: '/blog/2024-01-15-statecharts-whats-in-a-name/stately-style.png',
+      dark: '/blog/2024-01-15-statecharts-whats-in-a-name/stately-style-dm.png',
+    }}
+  />
+</p>
+
+In the Details panel of each machine in the Studio you can now find a set up controls for configuring and applying styles to the following elements of your statechart: states, events, actors, actions, and guards
+
+Naming styles include
+
+- Title Case
+- Sentence case
+- UPPER_CASE
+- PascalCase
+- camelCase
+- kebab-case
+- snake_case
+- lower case
+- dot.case \*
+- case.dot.reverse \*
```

Comment:
Not recommended because of wildcard transitions in SCXML/XState only working as a suffix

```suggestion
```